extensions/core/interfaces/icons/readonly.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+    <div>
+        <i class="material-icons">{{ value }}</i>
+    </div>
+</template>
+
+<script>
+import mixin from "../../../mixins/interface";
+import icons from "./icons.json";
+
+export default {
+  mixins: [mixin],
+  computed: {
+    icons() {
+      return icons;
```

Comment:
Icons isn't used in the component, but you do import the whole 1000 lines of JSON which increases the file size quite considerably